charts/tiphys/templates/app.yaml

```diff
@@ -115,6 +115,14 @@ spec:
         - name: {{ $.Chart.Name }}
           image: {{ default $.Values.defaultImage $app.image | quote }}
           imagePullPolicy: {{ default "Always" $app.service.imagePullPolicy | quote }}
```

Comment:
Add the following to https://github.com/opszero/tiphys/blob/main/charts/tiphys/values.yaml#L5

```
defaultSecurityContext:
  allowPrivilegeEscalation: false
  runAsUser: 1001
  runAsNonRoot: true
  privileged: false
```